sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/NavBarSteps.java

```diff
@@ -374,5 +374,12 @@ public NavBarSteps(WebDriverHelpers webDriverHelpers) {
               break;
           }
         });
+
+    When(
+        "I click on the Shares button from navbar",
+        () -> {
+          webDriverHelpers.waitForPageLoaded();
```

Comment:
this won't make any difference, is checking if DOm is ready at JS level. You can replace it to wait after element to be visible and clickable